airbyte-webapp/src/utils/dataPrivacy.ts

```diff
@@ -0,0 +1,75 @@
+declare global {
+  interface Window {
+    Osano?: {
+      cm: {
+        mode: "production" | "debug";
+        showDrawer: (type: string) => void;
+      };
+    };
+  }
+}
+
+const GDPR_TIMEZONES = [
+  "Africa/Ceuta",
+  "Asia/Famagusta",
+  "Asia/Nicosia",
+  "Atlantic/Azores",
+  "Atlantic/Canary",
+  "Atlantic/Madeira",
+  "Europe/Amsterdam",
+  "Europe/Athens",
+  "Europe/Berlin",
+  "Europe/Bratislava",
+  "Europe/Brussels",
+  "Europe/Bucharest",
+  "Europe/Budapest",
+  "Europe/Busingen",
+  "Europe/Copenhagen",
+  "Europe/Dublin",
+  "Europe/Helsinki",
+  "Europe/Lisbon",
+  "Europe/Ljubljana",
+  "Europe/Luxembourg",
+  "Europe/Madrid",
+  "Europe/Malta",
+  "Europe/Paris",
+  "Europe/Prague",
+  "Europe/Riga",
+  "Europe/Rome",
+  "Europe/Sofia",
+  "Europe/Stockholm",
+  "Europe/Tallinn",
+  "Europe/Vienna",
+  "Europe/Vilnius",
+  "Europe/Warsaw",
+  "Europe/Zagreb",
+];
```

Comment:
Just curious: is browser timezone a sufficient way to enforce GDPR? Seems a bit unreliable to me, but I don't have a better suggestion.

Reply:
I agree, this is not a really foolproof way. Luckily this is really only used to determine the default state of the newsletter flag and such I think it's an "okay enough" approach, without requiring to get a third party geo location service in place (which would be the better approach detecting by IP location). Osano itself for showing the right consent banner actually uses a proper IP based approach, so it's really only for the default state of this one switch we're needing to use this weaker approach.